### ISSUE TITLE
1.20.4 Version

### ISSUE THREAD

**Issue by IonicEcko** [Steveplays28/realisticsleep#56]
I may be doing something silly, but is there a way to get this working with 1.20.4? At the moment its throwing a compatibility error on launch.

**Comment by Steveplays28** [comment]
You could try using Fabric/Quilt [dependency overrides](https://fabricmc.net/wiki/tutorial:dependency_overrides) to change the supported Minecraft version, or edit the `jar` directly.
I'll test compatibility and update all my mods when I have time!

Thanks for checking out Realistic Sleep.

**Comment by ryu0q** [comment]
## **Here's how to modify the fabric_loader_dependencies.json file to make the RealisticSleep mod compatible with Minecraft 1.20.4:**

### **1. Locate the file:**

Navigate to your Minecraft or server installation's config folder. The typical path is .minecraft/config.
If the fabric_loader_dependencies.json file doesn't exist, create a new empty file with that name.

### **2. Open the file:**

Use a text editor like Notepad, Notepad++, VS Code, or any other plain text editor to open the fabric_loader_dependencies.json file.

### **3. Add the override:**

Paste the following code into the file, replacing any existing content:
```
{
  "version": 1,
  "overrides": {
    "realisticsleep": {
      "depends": {
        "minecraft": "1.20.4"
      }
    }
  }
}
```
### **4. Save the file:**
Make sure to save the changes you made to the fabric_loader_dependencies.json file.

## Explanation:

* The version key ensures the file is in the correct format.
* The overrides section holds modifications to mod dependencies.
* Inside overrides, you specify the mod's ID (realisticsleep), and within that, you define its new dependency on Minecraft 1.20.4 using the depends key.

## Important note:

While this method might allow the mod to function, there's no guarantee of full compatibility or stability, as it hasn't been officially tested with 1.20.4

**Comment by Steveplays28** [comment]
I've released a 1.20.3-1.20.4 version of Realistic Sleep with [`v1.10.2`](https://github.com/Steveplays28/realisticsleep/releases).
